autometa/config/project.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python
```

Comment:
This file does not appear to exist in the latest commit in `dev`, so you should delete the file here to avoid re-adding it upon merge.

Reply:
This could also be solved with an upstream pull from `dev`, so you could try that first because it will be easier.